Detections/SigninLogs/GitHub Activities from Infrequent Country.yaml

```diff
@@ -3,6 +3,10 @@ name: GitHub Activites from a New Country
 description: |
   'Detect activities from a location that was not recently or was never visited by the user or by any user in your organization.'
 severity: Medium
+requiredDataConnectors:
```

Comment:
This should not be included, Detections do not support CustomConnectors, it must be from the list or left out if it does not exist - [Query Style guide - Data connectors](https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#requireddataconnectors)